velox/functions/prestosql/tests/DateTimeFunctionsTest.cpp

```diff
@@ -2858,6 +2866,17 @@ TEST_F(DateTimeFunctionsTest, dateParse) {
       dateParse("116", "%y+"), "Invalid format: \"116\" is malformed at \"6\"");
 }
 
+TEST_F(DateTimeFunctionsTest, currentTimeTest) {
+  auto emptyRowVector = makeRowVector(ROW({}), 1);
+  auto tz = "America/Los_Angeles";
+  setQueryTimeZone(tz);
+  auto currentTimeBefore = getCurrentTime(tz);
+  auto result = evaluateOnce<std::string>("current_time", emptyRowVector);
```

Comment:
Does emptyRowVector need to be a variable ? Can the empty row be constructed in place here ?

Reply:
Changed it to be constructed in place in the function call